use

```diff
@@ -33,8 +33,8 @@ download() {
     echo "Unable to get tbls version." >&2
     exit 1
   }
-  rm -f "$TBLS_ARCHIVE"
-  curl -s -L -o "$TBLS_ARCHIVE" \
+  curl `if [ -f "$TBLS_ARCHIVE" ]; then echo -z "$TBLS_ARCHIVE"; fi` \ # Skip downloading if the newest file was downloaded before
```

Comment:
In my environment(bash), commenting after `\` break the command.

```bash
11:23 $ curl `if [ -f "$TBLS_ARCHIVE" ]; then echo -z "$TBLS_ARCHIVE"; fi` \ # Skip downloading if the newest file was downloaded before
curl: (6) Could not resolve host:
curl: (6) Could not resolve host: Skip
curl: (6) Could not resolve host: downloading
curl: (6) Could not resolve host: if
curl: (6) Could not resolve host: the
curl: (6) Could not resolve host: newest
curl: (6) Could not resolve host: file
curl: (6) Could not resolve host: was
curl: (6) Could not resolve host: downloaded
curl: (6) Could not resolve host: before
```

Reply:
I got it fixed at https://github.com/k1LoW/tbls/pull/426